packages/vega-scenegraph/src/util/svg/transform.js

```diff
@@ -14,8 +14,13 @@ export function translateItem(item) {
   return translate(item.x || 0, item.y || 0);
 }
 
+export function rotateItem(item) {
+  return translate(item.x || 0, item.y || 0)
+    + (item.angle ? ' ' + rotate(item.angle) : '');
+}
+
 export function transformItem(item) {
   return translate(item.x || 0, item.y || 0)
     + (item.angle ? ' ' + rotate(item.angle) : '')
-    + (item.scaleX || item.scaleY ? ' ' + scale(item.scaleX || 1, item.scaleY || 1) : '');   
+    + (item.scaleX || item.scaleY ? ' ' + scale(item.scaleX || 1, item.scaleY || 1) : '');
```

Comment:
Can `scaleX` or `scaleY` ever be 0 on purpose? Maybe it could happen when you want to animate changes.

Reply:
That could be a useful addition if/when animation is added, but should be the subject of a separate issue / PR.